packages/origin-backend/src/pods/user/user.entity.ts

```diff
@@ -47,9 +47,9 @@ export class User extends ExtendedBaseEntity implements IUser {
     @Column({ default: 0, nullable: false })
     rights: number;
 
-    @Column()
+    @Column({ default: 0 })
```

Comment:
Can we use the enum instead of 0?

Reply:
I think we might, just will need to rename the Enums because some use spaces in their names.

Reply:
for Column can add { default: 0, nullable: false }